components/brave_shields/common/features.cc

```diff
@@ -25,6 +25,10 @@ const base::Feature kBraveAdblockCnameUncloaking{
 // iframes that initiate a blocked network request.
 const base::Feature kBraveAdblockCollapseBlockedElements{
     "BraveAdblockCollapseBlockedElements", base::FEATURE_ENABLED_BY_DEFAULT};
+// When enabled, Brave will display a bubble inviting the user to turn on the
+// "Easylist-Cookie List" filter.
+const base::Feature kBraveAdblockCookieListBubble{
```

Comment:
BraveAdblockCookieListOptInBubble for consistency

Reply:
or just `BraveAdblockCookieListOptIn` would also be ok

Reply:
Done.